### PR TITLE
Issue template

### PR DESCRIPTION
**Pull request by tiokim**
Please check this PR from https://github.com/tiokim/osori-db.github.io/issues.
You can choose `Bug report` and  `OSS Contribution` once you press the `New Issue` button.

### CODE REVIEW

**Review by somang-park — APPROVED** [review]
LGTM :)

File: .github/ISSUE_TEMPLATE/Suggestion.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: Suggestion
+about: Suggest new features or an idea for OSORI
+title: ''
+labels: review required
```
Comment (MoonkiHong) [inline .github/ISSUE_TEMPLATE/Suggestion.md:5]:
```suggestion
labels: suggestion, review required
```
What about adding a label "suggestion" for this type of issue registration?

File: .github/ISSUE_TEMPLATE/oss_contribution.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+name: OSS Contribution
```
Comment (MoonkiHong) [inline .github/ISSUE_TEMPLATE/oss_contribution.yml:1]:
```suggestion
name: Data Contribution
```
What about modifying the name to "Data Contribution"?

File: .github/ISSUE_TEMPLATE/oss_contribution.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+description: Add or Modify OSS Information to OSORI
```
Comment (MoonkiHong) [inline .github/ISSUE_TEMPLATE/oss_contribution.yml:2]:
```suggestion
description: Add or Modify OSS Data Information to OSORI
```
What about modifying the description like above?

File: .github/ISSUE_TEMPLATE/oss_contribution.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+labels: ["oss", "review required"]
```
Comment (MoonkiHong) [inline .github/ISSUE_TEMPLATE/oss_contribution.yml:3]:
```suggestion
labels: ["data contribution", "review required"]
```
What about modifying the title of "oss" lable to "data contribution"?

**Review by MoonkiHong — COMMENTED** [review]
Please take a look at my opinion and let me know your feedback. 🙏

File: .github/ISSUE_TEMPLATE/oss_contribution.yml (hunk 2 above)
Comment (tiokim) [reply .github/ISSUE_TEMPLATE/oss_contribution.yml]: Done

File: .github/ISSUE_TEMPLATE/Suggestion.md (hunk 1 above)
Comment (tiokim) [reply .github/ISSUE_TEMPLATE/Suggestion.md]: Done

File: .github/ISSUE_TEMPLATE/oss_contribution.yml (hunk 4 above)
Comment (tiokim) [reply .github/ISSUE_TEMPLATE/oss_contribution.yml]: Done

File: .github/ISSUE_TEMPLATE/oss_contribution.yml (hunk 3 above)
Comment (tiokim) [reply .github/ISSUE_TEMPLATE/oss_contribution.yml]: Done

**Review by MoonkiHong — APPROVED** [review]
Looks good to me. Thanks for accepting my opinion. 😄

**Pull request merged**